diff --git a/node_modules/metaballs-js/src/shaders.js b/node_modules/metaballs-js/src/shaders.js
index db1b4b2..d410606 100644
--- a/node_modules/metaballs-js/src/shaders.js
+++ b/node_modules/metaballs-js/src/shaders.js
@@ -44,6 +44,7 @@ void main(){
     float xMultiplier = windowSize.x / 100.0;
     float yMultiplier = windowSize.y / 100.0;
     
+
     float x = gl_FragCoord.x;
     float y = gl_FragCoord.y;
     float v = 0.0;
@@ -61,6 +62,7 @@ void main(){
     if (v > 1.0) {
         gl_FragColor = vec4(${colorVec4.join(', ')});
     } else {
+        gl_FragColor = vec4(x/windowSize.x, 0, 0, 1);
         gl_FragColor = vec4(${backgroundColorVec4.join(', ')});
     }
 }
